src/services/person.ts

```diff
@@ -0,0 +1,71 @@
+import AWS from 'aws-sdk';
+
+import { tPerson, tError } from '../model/contract';
+
+import { getRandomData } from './randamData';
+
+async function getPersonList(): Promise<{data: Array<tPerson> | null, error: tError | null}> {
+  const dynamoDb = new AWS.DynamoDB.DocumentClient();
+  const params = { TableName: process.env.DYNAMODB_PERSON_TABLE ? process.env.DYNAMODB_PERSON_TABLE : '' };
+
+  try {
+    const items = await dynamoDb.scan(params).promise();
+    return { data: <Array<tPerson>>items.Items, error: null };
+  } catch (error: any) {
+    console.error("Error while creating new person. Params:", params, " Details:", error);
+    return { data: null, error: { code: 500, detail: 'Internal error while getting persons list' }};
```

Comment:
Não eh comum retornar um código de erro pq não eh uma chamada http. O próprio retorno da função, ou lançamento de exceção já indica se foi ok ou com problema.

Reply:
Eu reutilizei os código http porque no final é o que vai ser retornado no endpoint, mas daria pra criar códigos internos com constantes, acho que seria o ideal, e ai colocar o código http no handler. Não estou usando exceção porque já estou seguindo mais ou menos um padrão que o Tiago criou para evitar usar try/catch https://stutzlab.atlassian.net/wiki/spaces/SL/pages/1507329/Tratamento+de+erros

Reply:
To sem acesso nesse arquivo. Se puder manda o conteúdo por aqui pra eu ver. Vamos tentar seguir ele sim.

Reply:
Vou mandar no WhatsAPP